setup.cfg

```
@@ -33,6 +33,7 @@ install_requires =
     audiofile >=1.0.0
     audobject >=0.5.0
     audresample >=0.1.6
+    filelock
```

Just to be on the safe side:

```suggestion
    filelock >=3.6.0
```